stats/influxdb/collector.go

```diff
@@ -108,10 +108,12 @@ func (c *Collector) commit() {
 	}
 
 	for _, sample := range samples {
+		tags := sample.Tags.CloneTags() //TODO: optimize when implementing https://github.com/loadimpact/k6/issues/569
```

Comment:
The idea with this TODO is that since [this pull request](https://github.com/loadimpact/k6/pull/578), `sample.Tags` is now comparable, it could be used as a map key for creating a simple cache. If you had something like this outside of the `for` loop:
```go
cache := map[sample.Tags]struct {
	tags   map[string]string
	values map[string]interface{}
}{}
```
In the `for` loop you'd be able to check if certain tags are already in the cache with 
```go
if cached, ok := cache[sample.Tags]; ok {
	// Reuse the cached.tags, create a new map with {"value": sample.Value} and copy all elements from cached.values to it
} else {
	// Do what you currently do and then save the resulting tags and values as a new entry in the cache
}
```
I'm fine with merging this as it is, if you don't want to bother with this probably premature optimization. If it turns out that it's an issue, we'll optimize it in the future.

Reply:
I'm looking at this now so we can hold off with the merge.